sys/shell/commands/sc_cord_ep.c

```diff
@@ -32,8 +32,12 @@ static int make_sock_ep(sock_udp_ep_t *ep, const char *addr)
     if (sock_udp_str2ep(ep, addr) < 0) {
         return -1;
     }
+    /* if netif not specified in addr */
+    if ((ep->netif == SOCK_ADDR_ANY_NETIF) && (gnrc_netif_numof() == 1)) {
+        /* assign the single interface found in gnrc_netif_numof() */
+        ep->netif = (int16_t)gnrc_netif_iter(NULL)->pid;
```

Comment:
Don't include my deleted errors ;-)
```suggestion
        ep->netif = (uint16_t)gnrc_netif_iter(NULL)->pid;
```

Reply:
I thought pid is kernel_pid_t which is [typedef of int16_t](http://api.riot-os.org/kernel__types_8h.html#a8375139300d7cbf23bd8bd89ddddbe84)?

Reply:
okay nvm, `ep->netif` is `uint16_t`.